documentation/Install_PACTA_as_a_first-time_user

```diff
@@ -0,0 +1,114 @@
+- - -
+## Overview:
+This how-to describes setting up and running a test PACTA analysis for a first time user and how to generate interactive reports.
+
+**Keywords:**
+
+## Before you start:
+### System requirements:
+|           | Recommended   | Minimum               |
+|-----------|---------------|-----------------------|
+| RAM       | 16 GB or more | 8 GB                  |
+| Windows   | 64-bit        | 32-bit                |
+| Mac OS    |               |                       |
+| Linux     |               |                       |
+| R version | 3.6.1         | no earlier than 3.4.3 |
+
+### Software Requirements:
+* GitHub (ex. [GitHub Desktop](https://desktop.github.com/))
+* A GitHub account added to the [2DegreesInvesting GitHub](https://github.com/2DegreesInvesting) organization
+* [RStudio (recommended version 1.1.456)](https://rstudio.com/products/rstudio/)
+* MikTeX- a distribution of the LaTeX typesetting system; includes TeXworks (preferably version 2.9)
+
+### Required libraries and packages:
+—command for installing binaries—
+Please review and consolidate. Current documentation states:
+> “Necessary packages are: tidyr, dplyr, scales, reshape2, tidyverse, readxl, tidyselect (and dependencies). Further R packages needed: grid, ggplot2, ggthemes, dplyr, reshape2, gridExtra, scales, stringr, extrafont, tidyr, knitr, RColorBrewer, matrixStats, rworldmap, ggmap, cowplot, ggrepel, readxl, tidyverse, ggforce, sitools , countrycode.”
+
+- - -
+## How to run a PACTA analysis with a test portfolio:
+### Step 1 — Create a local copy of the repositories:
+It is recommended to clone both repositories into the **same local directory**.
+
+#### To create a local copy:
+1. [Fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) the [PACTA_analysis](https://github.com/2DegreesInvesting/PACTA_analysis) repository
+2. Fork the [PACTA-data](https://github.com/2DegreesInvesting/PACTA-data) repository
+3. From your Git software [clone](https://docs.github.com/en/get-started/quickstart/fork-a-repo#cloning-your-forked-repository) the public PACTA_analysis repository
+4. Clone the PACTA_data private repository
+
+> Every time you work with the local copy, [pull](https://docs.github.com/en/get-started/using-git/getting-changes-from-a-remote-repository) updates through your method of Git to ensure you’re working with the most recent version. Check that the branch you’re working on is the master branch—unless otherwise required.
+
+### Step 2 — Launch RStudio:
+To launch RStudio with the PACTA_analysis files:
+* From your local directory open` PACTA_analysis > R > PACTA_analysis.rproj `
+
+### Step 3 — Running web_tool_script1 and web_tool_script2:
+The scripts use the following example files located in `working_directory`:
+* `20_Raw_Inputs > TestPortolio_Input.csv`
+* `10_Parameter_File > TestPortfolio_Input_PortfolioParameters.yml`
+	* The `portfolio_name_in` variable must match the `.csv` file name
+
```

Comment:
Between steps 2 and 3, we are missing an explanation of how to install dependencies. 
@meggpie this did not have a clear solution while we were chatting with you, so not meant to sound as a criticism of this draft, but I think this needs to get solved before we complete this @cjyetman @AlexAxthelm

Reply:
Relates to the conversation here: https://github.com/2DegreesInvesting/PACTA_analysis/pull/514#issuecomment-977806196
The issue here: #466
etc.

Reply:
@cjyetman I just want to follow up on this PR. Regarding my points above, should the Step between Step 2 and 3 be "install all packages listed in `README`?"

Reply:
that seems reasonable to me

Reply:
added that step, just needs instructions on how to install if someone could provide or chat with me sometime next week @jdhoffa @cjyetman @AlexAxthelm